packages/stac/lib/src/parsers/widgets/stac_image_filter/stac_image_filter_parser.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:typed_data';
+import 'dart:ui';
+
+import 'package:stac/src/parsers/widgets/stac_double/stac_double.dart';
+import 'package:stac/src/parsers/widgets/stac_image_filter/stac_image_filter.dart';
+
+class StacImageFilterParser {
+  static ImageFilter? parse(StacImageFilter? filter) {
```

Comment:
Can we please turn this into an extension method?

https://dart.dev/language/extension-methods